sdk/cosmos/azure-cosmos/azure/cosmos/_base.py

```diff
@@ -679,6 +679,7 @@ def validate_cache_staleness_value(max_integrated_cache_staleness):
 
 
 def _stringify_auto_scale(offer):
+    # type: (Dict[str, Any]) -> Any
```

Comment:
Please use Python 3 type hints exclusively. We no longer support Python 2 at all and thus don't want to proliferate Python 2 type hints.

https://docs.python.org/3/library/typing.html

Reply:
fixed!